Assets/Plugins/Source/EOS_SDK/Core/Utf8String.cs

```diff
@@ -11,6 +11,8 @@ namespace Epic.OnlineServices
 	[System.Diagnostics.DebuggerDisplay("{ToString()}")]
 	public sealed class Utf8String
 	{
+		static public Utf8String EmptyString = new Utf8String();
```

Comment:
Can we lock this down to be readonly, or have only a get-accessor?

I've had a project before where "Vector3.zero" was set in one class and re-used, and it took a while to debug a problem when it was realized that the reference of the returned value was being modified rather than using it by value.